README.rst

```diff
@@ -20,12 +57,13 @@ guideline.
 
 
 
+All data deposited here have been either
```

Comment:
The texts on lines 62-64 seem to be a  quotation block. (Check how it is rendered now on github.) Did you do it on purpose?